core/src/main/java/org/testcontainers/utility/DockerImageName.java

```diff
@@ -2,28 +2,29 @@
 
 
 import com.google.common.net.HostAndPort;
-import lombok.AccessLevel;
-import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 import java.util.regex.Pattern;
 
 @EqualsAndHashCode(exclude = "rawName")
-@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
Might reinstate, but Lombok seemed to be choking and was getting in the way.

Reply:
Just curious: what was(is ?) the problem?

Reply:
Not sure I remember the specifics, but I think it was the IDE not seeing the generated constructor (doubtless plugin related)

Reply:
ah, okay, seems to be https://youtrack.jetbrains.com/issue/IDEA-248146 then (fixed in the latest IDEA (rc?))

Reply:
Yep, it's working now, and I've restored `@AllArgsConstructor`.